src/Object/Client/ClientResource.php

```diff
@@ -34,6 +36,28 @@ class ClientResource extends BaseApiResource
         self::PRIVATE_LINKS_VIEWERS => 'Private Link Viewers'
     ];
 
+
+    /**
+     * @return array
+     */
+    public function getData()
+    {
+        $data = parent::getData();
+        for ($i = 0; $i < count($data['resources']); $i++) {
```

Comment:
Can you just unset stuff in a foreach in php? In java you get a ConcurrentModificationException.